src/Pages/miyoungshin/Main/Main.js

```diff
@@ -1,8 +1,74 @@
 import React from 'react';
+import CommonNav from '../../../Components/Nav/Nav';
+import HandleFeed from './Component/FeedList';
+import FEED from './Data/FeedData';
+import './Main.scss';
 
 class MainMi extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedsList: [],
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      feedsList: FEED,
+    });
+  }
+
```

Comment:
```suggestion
  constructor() {
    super();
    this.state = {
      feedsList: FEED,
    };
  }
```
불필요한 렌더가 일어날 것 같은데 초기값으로 써주는 건 어떨까요?

Reply:
수정했습니다